### ISSUE TITLE
<c> XML documentation contents should be placed on same line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#514]
In contrast to #513, we should issue a warning if the contents of the `<c>` XML documentation is placed on another line.

Example:
```c#
/// <c>
/// some text
/// </c>
```
should issue a warning.

It should be fixed into:
`<c>some text</c>`
